src/schema/v1.json

```diff
@@ -229,7 +229,7 @@
      "examples": [
        "31.20200127.20.1"
       ],
-      "pattern":"[0-9]{2}\\.[0-9]{8,}\\.[0-9]{2}\\.[0-1]"
+     "pattern":"^(?!\\s*$).+"
```

Comment:
LGTM - this now matches the value that is set for `ostree-version` as well

Reply:
IMHO, weakening the pattern is not _great_, but in light of the fact that all the other patterns are nothing more than `not NULL`, this having stricter patterns sounds like a new version of the schema.

Reply:
See also discussions at https://github.com/coreos/mantle/pull/1175#discussion_r376189220 related to this.